abe/resource_models/event_resources.py

```diff
@@ -133,6 +134,12 @@ def post(self):
         new_event = db.Event(**received_data)
         if new_event.labels == []:  # if no labels were given
             new_event.labels = ['unlabeled']
+        else:
+            for label in new_event.labels:
+                label_info = label_resources.LabelApi().get(id=label)
+                if type(label_info) is dict and 'protected' in label_info:
+                    if label_info['protected']:
```

Comment:
You can combine `'protected' in label_info && label_info['protected']` into a single test `label_info.get('protected', False)`. `label_info.get` is a variant of `label_info[…]` that returns a default value, instead of failing, when the key isn't in the dictionary.

The first version looks up 'protected' twice; the second one, only once. That doesn't matter much here — it's not performance-critical code, and the dictionary is small.